### PR TITLE
PR: Rewrote configuration to emphasis singular `App` class usage

### LINKED ISSUE #231: DWCJCLASSNAME in config.bbx is no longer supported
Remove this section
https://documentation.webforj.com/docs/getting_started/configuration#editing-the-bbj-config-file

https://github.com/webforj/webforj/pull/726

### PR DESCRIPTION
**Pull request by bbrennanbasis**
This PR closes #231.

### CODE REVIEW

File: docs/getting-started/configuration.md [hunk 1]
```diff
@@ -38,29 +38,43 @@ When using the installation plugin, the tags within the `<configuration>` tag ca
...
 
-- **`<classname>`** This tag should contain the package and class name of the application you wish to run. This is helpful for projects that contain multiple classes that extend the `App` class, and allows you to choose which program to run from the base URL.
+- **`<classname>`** This tag should contain the package and class name of the application you wish to run. This will be the single class in your project that extends the `App` class and runs from the base URL.
 
 - **`<publishname>`** This tag specifies the name of the application in the published URL. Generally, to run your program, you'll navigate to a URL similar to `http://localhost:8888/webapp/<publishname>`, replacing `<publishname>` with the value in the `<publishname>` tag. Then, the program specified by the `<classname>` tag is run.
 
 - **`<debug>`** The debug tag can be set to true or false, and will determine whether or not the browser's console displays error messages thrown by your program. 
 
-### Running a specific application
+### Running a specific program
```
Comment (hyyan) [inline docs/getting-started/configuration.md:47]: @MatthewHawkins I believe this section is misplaced and doesn't belong in the configuration section. Additionally, routing is optional anyway. what do you think ?

**Review by hyyan — CHANGES_REQUESTED** [review]

**Comment by hyyan** [comment]
@bbrennanbasis It looks good. Please review all the Vale issues on this page and ensure they are fixed. Thank you!

**Comment by bbrennanbasis** [comment]
@hyyan I removed the specific step-by-step instructions for routing and mentioned the other way of running a specific app (using the class that extends `App`).

**Comment by hyyan** [comment]
@MatthewHawkins, please review this and check if it aligns with the plans for documenting the new configuration approach. It might be best to hold off until you've completed that.

**Comment by MatthewHawkins** [comment]
@hyyan I read over the changes here and it looks better to me. I'm not opposed to merging before the new WAR configuration is documented, as I believe enforcing this behavior with the routing changes implemented is sufficient, but if you'd prefer to wait until that's completed, then no problem.

**Review by hyyan — CHANGES_REQUESTED** [review]

**Comment by hyyan** [comment]
@bbrennanbasis requested minor formatting changes
@MatthewHawkins merge when ready please

**Comment by bbrennanbasis** [comment]
@hyyan 
- Changed application to app
- Removed misleading wording talking about with and without the plugin
- Rewrote Default class section and changed the header name

**Review by MatthewHawkins — APPROVED** [review]

**Pull request merged**